lib/net.js

```diff
@@ -7,8 +7,10 @@ const util = require('util');
 const assert = require('assert');
 const cares = process.binding('cares_wrap');
 const uv = process.binding('uv');
-const Pipe = process.binding('pipe_wrap').Pipe;
 
+const TTY = process.binding('tty_wrap');
```

Comment:
The naming suggests it's the TTY constructor but it's really the tty_wrap object (which has the constructor as a property.)